src/scripts/Update.ts

```diff
@@ -800,13 +800,13 @@ class Update implements Saveable {
         },
 
         '0.9.8': ({ playerData, saveData, settingsData }) => {
-            saveData.oakItemLoadouts = saveData.oakItemLoadouts.map((list, index) => ({ name: `Loadout ${index + 1}`, loadout: list }));
+            saveData.oakItemLoadouts = saveData.oakItemLoadouts?.map((list, index) => ({ name: `Loadout ${index + 1}`, loadout: list }));
 
             // Fix pokerus
             saveData.party.caughtPokemon.forEach(p => {
                 let status = (p[8]) ? 2 : 0;
                 const requiredForCured = saveData.challenges.list.slowEVs ? 5000 : 500;
-                if (saveData.statistics.effortPoints[p.id] >= requiredForCured) {
+                if (saveData.statistics.effortPoints?.[p.id] >= requiredForCured) {
```

Comment:
And same thing for this? (although not sure if it's a `{}` or `[]`)
```suggestion
                saveData.statistics.effortPoints = saveData.statistics.effortPoints || {};
                if (saveData.statistics.effortPoints.[p.id] >= requiredForCured) {
```

Reply:
I don't think a default is needed here since we are moving effortpoints onto the pokemon anyway